### PR TITLE
How to Start and Run a Jupyter Notebook

### LINKED ISSUE #2: How to start and run a notebook

### PR DESCRIPTION
**Pull request by julienchastang**
This closes #2.

### CODE REVIEW

File: notebook.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+Enter some code in the first cell:
+
+    print("hello world")
+
+and press Shift-Enter to actually run the cell yielding
+
+    hello world
+
+<img src="images/jupyter.gif" alt="Starting Jupyter" onclick='this.src=this.src'/>
```
Comment (dopplershift) [inline notebook.md:41]: Let's avoid HTML if we don't need it--unless there's something magical about the onclick handler? Markdown can do images:

```
![Starting Jupyter](images/yaktocat.png)
```

File: notebook.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+title: How to Start and Run a Jupyter Notebook
+---
+
+# How to Start and Run a Jupyter Notebook
+
+As discussed in the [introduction](http://unidata.github.io/online-python-training/introduction.html), Jupyter notebooks provide a tremendous advantage for scientific computing, analysis and visualization. Here, we will explain how to start a Jupyter notebook. We will present just enough information to get you started. For a more complete treatment of Jupyter notebook software, please read the [official documentation](https://jupyter-notebook.readthedocs.org/en/latest/).
```
Comment (dopplershift) [inline notebook.md]: Would relative links be a better idea than absolute URLs?

File: notebook.md (hunk 1 above)
Comment (julienchastang) [inline notebook.md:41]: Yes, actually on the `onclick` is important to restart the video if the viewer wants. Can this be handled in pure MD?

File: notebook.md (hunk 2 above)
Comment (julienchastang) [inline notebook.md]: Agreed.

File: notebook.md (hunk 1 above)
Comment (dopplershift) [inline notebook.md:41]: Interesting, I hadn't seen that before. I suspect HTML is the right tool then. Could you add an HTML comment there so we know why it's there for the future?

Comment (julienchastang) [inline notebook.md:41]: Yes. (I agree it is nice to be able to iterate on PRs herein. github, travis, etc. handles all this nicely.)

**Comment by julienchastang** [comment]
The only check that is failing at this point is the `conda-windows.html` forthcoming link.

**Pull request merged**